src/canvas.py

```diff
@@ -267,7 +267,7 @@ def set_legend(self):
             if labels:
                 self.top_right_axis.legend(
                     new_lines, labels,
-                    loc=self.application.plot_settings.legend_position,
+                    loc=self.application.plot_settings.legend_position.lower(),
```

Comment:
Subtle, but indeed cleaner to do this here in canvas.

Reply:
The reason I've moved it to here is, that previously it was in the huge block of code, that loads and saves the preferences to ui.
The alternative was making an exception there and handling this key manually, but I though this was cleaner. Config keys should work just fine, as "best" isn't an option anymore, but "Best" is still the first choice.